daisy_integration_tests/v2v_adapt_windows_byol.wf.json

```diff
@@ -0,0 +1,92 @@
+{
+  "Name": "v2v-adapt",
+  "Vars": {
+    "image_name": {
+      "Required": true,
+      "Description": "The name of the imported image."
+    },
+    "source_disk": {
+      "Required": true
+    },
+    "worker_disk": {
+      "Value": "worker-disk"
+    }
+  },
+  "Steps": {
+    "create-disk": {
+      "CreateDisks": [
+        {
+          "Name": "${worker_disk}",
+          "SourceImage": "projects/v2v-dev/global/images/family/v2v-candidate",
+	        "SizeGb": "200",
+          "Type": "pd-standard",
+          "Licenses": [
+            "https://compute.googleapis.com/compute/v1/projects/vm-options/global/licenses/enable-vmx"
```

Comment:
To enable nested KVM, you also need to specify a CPU platform of haswell of newer. Either switching to n2 instances, or specifying [minCpuPlatform](https://cloud.google.com/compute/docs/reference/rest/v1/instances) in your instance definition will achieve that.

I usually use minCpuPlatform, since I've observed quota issues in my test account when using n2.

https://cloud.google.com/compute/docs/instances/enable-nested-virtualization-vm-instances

Reply:
Specified minCpuPlatform as "Intel Haswell"